src/Modal/Modal.js

```diff
@@ -91,7 +91,7 @@ export function Modal({ children, title, animationDuration, showClose, onClose,
   };
 
   const scrollToTop = () => {
-    if (modalRef.current) {
+    if (modalRef.current && modalRef.current.scroll) {
```

Comment:
do we have optional chaining in `molekule`?

Reply:
not at the moment